openedx_tagging/core/tagging/api.py

```diff
@@ -484,3 +484,60 @@ def delete_tags_from_taxonomy(
     """
     taxonomy = taxonomy.cast()
     taxonomy.delete_tags(tags, with_subtags)
+
+
+def copy_tags(
+    source_object_id: str,
+    dest_object_id: str,
+    include_deleted: bool = True,
+    object_tag_class: type[ObjectTag] = ObjectTag
+):
+    """
+    Copy all tags from one object to another.
+
+    This keeps all not-copied tags and delete all
+    previous copied tags of the dest object.
+    If there are not-copied tags that also are in 'source_object_id',
+    then they become copied.
+    """
+    ObjectTagClass = object_tag_class
+
+    def object_has_tag(object_tag):
```

Comment:
Can we rename this to something like `dest_object_has_tag`?

Reply:
Updated: https://github.com/openedx/openedx-learning/pull/236/commits/6b73ec4ff4c282a007eea3fe231b9db351faf9a7